action-types.yml

```diff
@@ -0,0 +1,17 @@
+# See https://github.com/krzema12/github-actions-typing
+inputs:
+  vswhere-path:
+    type: string
+  vs-version:
+    type: string
+  vs-prerelease:
+    type: boolean
+  msbuild-architecture:
+    type: enum
+    name: Architecture
+    allowed-values:
+    - x86
+    - x64
```

Comment:
arm64 is also a valid option

Reply:
Added to `action.yml` and the types file

Reply:
Great example of action typings making the documentation clearer 😄